### ISSUE TITLE
[Feature] Material You / Dynamic colors support

### ISSUE THREAD

**Issue by starry-shivam** [leonlatsch/Photok#328]
**Is your feature request related to a problem? Please describe.**
Well, it's not strictly related to a problem, but it'll be a nice addition. Since most apps support dynamic themes already, it results in a consistent user experience, and I think it also looks quite nice.

**Describe the solution you'd like**
Adding dynamic/Material You theme support. 

**Describe alternatives you've considered**
N/A

**Additional context**
I can probably work on this and create a PR after getting the green signal.

**Comment by starry-shivam** [comment]
> not all of Photok is migrated to compose yet, so it will only apply to Gallery, Albums and the main menu for now.

Yeah, I realized it just now after taking a look at the source code. However, I think we can add a fallback Material You theme for the XML views to keep it consistent. I'll give it a try after #318 and see how it goes.

**Comment by leonlatsch** [comment]
Tried it. Didnt like it. Photok loses its identity. 

But I will update the material theme with the next update so it fits better in modern android
